src/hotel-management-app/src/app/services/cart.service.ts

```diff
@@ -0,0 +1,80 @@
+import { Injectable } from '@angular/core';
+import { CartItem } from '../models/cart.model';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class CartService {
+  private cartKey = 'cart';
```

Comment:
Could you explicitly assign types to the variable than auto typecasting

Reply:
resolved